.binder/requirements.txt

```diff
@@ -1,15 +0,0 @@
-itk-elastix>=0.17.1
```

Comment:
The requirements need to be here for mybinder / repo2docker to work.

Reply:
OK, let's remove then the `requirements.txt` that exists in the /examples directory because it is not kept in sync with the binder one.

Reply:
@N-Dekker for more information, see the repo2docker docs: https://repo2docker.readthedocs.io/en/latest/

Reply:
Ah, I see now, at https://repo2docker.readthedocs.io/en/latest/usage.html#usage-config-file-location 

> `repo2docker` will look for configuration files in:
> - A folder named `binder/` in the root of the repository.
> - A folder named `.binder/` in the root of the repository.
> - The root directory of the repository.

Thanks Matt. Found the direct link to this information with help from my colleague Baldur van Lew.